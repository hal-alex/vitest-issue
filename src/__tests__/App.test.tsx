import { render } from "@testing-library/react"
import App from "./../App"

describe("App", () => {
  test("should render correctly", () => {
    const { asFragment } = render(<App />)
    expect(asFragment()).toMatchSnapshot()
  })
})
