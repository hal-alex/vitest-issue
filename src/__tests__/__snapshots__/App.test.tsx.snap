// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App > should render correctly 1`] = `
<DocumentFragment>
 # the CSS classes should be defined here
 # for example
 .sc-braxZu {
  padding: 20px;
 }
  <div>
    <a
      href="https://vitejs.dev"
      target="_blank"
    >
      <img
        alt="Vite logo"
        class="logo"
        src="/vite.svg"
      />
    </a>
    <a
      href="https://react.dev"
      target="_blank"
    >
      <img
        alt="React logo"
        class="logo react"
        src="/src/assets/react.svg"
      />
    </a>
  </div>
  <h1>
    Vite + React
  </h1>
  <div
    class="card"
  >
    <button>
      count is 0
    </button>
    <p>
      Edit 
      <code>
        src/App.tsx
      </code>
       and save to test HMR
    </p>
  </div>
  <p
    class="read-the-docs"
  >
    Click on the Vite and React logos to learn more
  </p>
  <button
    # this CSS class should be defined on top
    class="sc-braxZu iJqifC"
  >
    Test Button
  </button>
</DocumentFragment>
`;
